euth_wagtail/wsgi.py

```diff
@@ -10,7 +10,9 @@
 import os
 
 from django.core.wsgi import get_wsgi_application
+from whitenoise.django import DjangoWhiteNoise
 
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "euth_wagtail.settings")
 
 application = get_wsgi_application()
+application = DjangoWhiteNoise(application)
```

Comment:
There is a [WhiteNoise Middleware](http://whitenoise.evans.io/en/stable/django.html#enable-whitenoise) you can use. So you don't have to modify `wsgi.py`. 😃